plop-templates/components/src/component.css

```diff
@@ -0,0 +1,4 @@
+/* General........................................................ */
+
+:host {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this empty rule. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=css%3Aempty-rules)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this usage of the experimental "host" pseudo. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=css%3Aexperimental-pseudo-usage)